syntax = "proto3";

package fixbug;

option cc_generic_services = true;
message ResultCode
{
    int32 errcode = 1;
    bytes errmsg = 2;
}


message GetFriendRequest  // Rpc 参数类型
{
    int32 userid = 1;
}

message GetFriendResponse // Rpc 返回值类型
{
    ResultCode result = 1;
    repeated bytes friends = 2; // repeated 表示重复的，即列表的数据
}



// 好友模块
service FriendServiceRpc // Rpc函数描述
{
    rpc GetFriendsList(GetFriendRequest) returns(GetFriendResponse);
}